code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -93,7 +93,7 @@
 	name = "Vitamin"
 	description = "All the best vitamins, minerals, and carbohydrates the body needs in pure form."
 	value = 0.5
-
+	nutriment_factor = 15 * REAGENTS_METABOLISM //The are the best food for you!
```

Comment:
Reminder that vitamin also somewhat improves food metabolism for a period of time, so it's already quite efficient.

Reply:
This is the same value has it was before. Im *not* nerfing this as is almost always made from cooks items and is to help balance lava-land healing  I.G